seungjong/.env.sample

```
@@ -0,0 +1,7 @@
+DB_CONNECTION = mysql
+DB_HOST = localhost
+DB_PORT = 3306
+DB_USERNAME = username
+DB_PASSWORD = password
+DB_DATABASE = database_name
+DB_LOGGING = TRUE
```

```suggestion
DATABASE_URL="rdbmsType://username:password@iphost:portNumber/databaseName"

PORT=server_port

DB_CONNECTION=db_type
DB_HOST=db_host
DB_PORT=db_port
DB_USERNAME=username
DB_PASSWORD=password
DB_DATABASE=database_name
DB_LOGGING=true or false
```
1. 환경변수에 들어가는 value들은 실제 값 대신 최대한 중의적인 표현을 활용하여 작성해주셔야 합니다. 외부에서 .env.sample을 보고 해당 api server의 database는 mysql을 쓰는 것을 알수 없게 해야합니다. mysql 고유의 port number인 3306 또한 마찬가지입니다.
2. app.js에서 PORT 라는 환경변수를 사용하고 있으므로 마찬가지로 .env.sample에 추가되어야겠습니다.
3. dbmate 연결 목적의 DATABASE_URL 또한 작성해주셔야합니다. 여기에는 개인이 로컬 환경에서 어떠한 rdbms를 쓰는지 그 힌트가 될 수 있는 단어와, 포트번호를 지우고 중의적인 단어를 대신 사용해주세요.